### PR TITLE
docs: improve ISSUE_TEMPLATEs [skip ci]

### PR DESCRIPTION
**Pull request by WaterLemons2k**
# What does this PR do?
Improve the issue templates:

- Use `labels` rather than `title` to indicate the type of issue.
- Request to add a description of how to reproduce the bug in the **Problem description**.
- Select the DNS providers used.
- Move **Searched** below **Unsearched** to prevent **Searched** from being selected by default.
- Describes how to get the version of ddns-go.
- Make sure that the person requesting the **Feature request** is using the latest version.
- Disable blank issues.
- Let those who have questions ask them in the Discussions.

# Motivation

# Additional Notes
1. [议题表单的语法 - GitHub 文档](https://docs.github.com/zh/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)
2. [为仓库配置议题模板 - GitHub 文档](https://docs.github.com/zh/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
       label: 解决的问题
-      
+      description: 请描述此功能可以解决什么问题
+
   - type: textarea
     attributes:
       label: 附加信息
+      description: 对此功能的其它描述
+
+  - type: input
```
Comment (jeessy2) [inline .github/ISSUE_TEMPLATE/feature_request.yml:23]: 这里加一个版本，有何用处？

Comment (WaterLemons2k) [reply .github/ISSUE_TEMPLATE/feature_request.yml:23]: 这里加版本是为了避免有人提出旧版本中不存在但新版本中存在的功能。

Comment (jeessy2) [reply .github/ISSUE_TEMPLATE/feature_request.yml:23]: 感觉有点奇怪，需求一个新功能，又需要输入一个版本号，看换个什么词

Comment (WaterLemons2k) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: 确保提出功能的人使用的是最新版本。

**Pull request merged**